service/lambda/runtime/event/snsevt/definition.go

```diff
@@ -67,6 +67,14 @@ type Record struct {
 	// The URL to the certificate that was used to sign the message.
 	SignatureCertURL string
 
+	// The URL that you must visit in order to confirm the subscription.
+	// Alternatively, you can instead use the Token with the ConfirmSubscription action to confirm the subscription.
```

Comment:
This lines are too wide (line 74 too). It's 80characters max (can be more if there is a link line in line 64).
Please make the modification 💛